src/util/lang_string.ts

```diff
@@ -30,18 +30,50 @@ export const escapeXml = (string: string): string =>
  * @return {Array} array containing the graphemes
  */
 export const graphemeSplit = (textstring: string): string[] => {
-  const graphemes = [];
-  for (let i = 0, chr; i < textstring.length; i++) {
+  if ('Intl' in window && 'Segmenter' in Intl) {
+    const segmenter = new Intl.Segmenter(undefined, { granularity: 'grapheme' });
+    const segments = segmenter.segment(textstring);
+    return Array.from(segments).map(segment => segment.segment);
+  }
+
+  //Fallback
+  return _graphemeSplit(textstring);
+};
+
+const _graphemeSplit = (textstring: string): string[] => {
```

Comment:
Let's call this graphemeSplitImpl as we do for other internal methods, but let's keep it with the original behaviour.